### PR TITLE
Removal of unused code due to old <5.2 QT ibrary

### PR DESCRIPTION
**Pull request by mosfet80**
Thank you for creating a pull request to contribute to FreeCAD! Place an "X" in between the brackets below to "check off" to confirm that you have satisfied the requirement, or ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10) if there is something you don't understand.

- [ ]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

Please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [1.0 Changelog Forum Thread](https://forum.freecad.org/viewtopic.php?f=10&t=69438).

---

### CODE REVIEW

**Comment by luzpaz** [comment]
Do you mind specifying the Qt version number ?

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842">pipeline status</a> for feature branch [PR_8842](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842). Pipeline [803283416](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/803283416) was triggered at [65e7ed7](https://github.com/FreeCAD/FreeCAD/pull/8842/commits/65e7ed76c3610ce5e94d97dd7a842b2ed59a2049). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by mosfet80** [comment]
> Do you mind specifying the Qt version number ?
<5.2  

I also removed the verification checks for including libraries when qt version is >=5

File: src/Mod/Sandbox/Gui/Command.cpp [hunk 1]
```diff
@@ -1150,13 +1090,6 @@ CmdTestGDIWidget::CmdTestGDIWidget()
 
 void CmdTestGDIWidget::activated(int)
 {
-#ifdef Q_OS_WIN32
```
Comment (wwmayer) [inline src/Mod/Sandbox/Gui/Command.cpp:1153]: Your changes are OK except for this one. GDIWidget is a test class to see how Qt widgets can be combined with Win32's GDI system. As the name says the Sandbox workbench is only there to play with some exotic features of used libraries.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842">pipeline status</a> for feature branch [PR_8842](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842). Pipeline [803298497](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/803298497) was triggered at [c2f8a58](https://github.com/FreeCAD/FreeCAD/pull/8842/commits/c2f8a58ed7b1e407f2c18236e1a3ac62848de6c1). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842">pipeline status</a> for feature branch [PR_8842](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842). Pipeline [803300239](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/803300239) was triggered at [ecfed1b](https://github.com/FreeCAD/FreeCAD/pull/8842/commits/ecfed1b4dd475b4dfc6c5c230f5697350aafe19b). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842">pipeline status</a> for feature branch [PR_8842](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842). Pipeline [803301116](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/803301116) was triggered at [ee70181](https://github.com/FreeCAD/FreeCAD/pull/8842/commits/ee701815d8321035bcdf50c17fdd79f66b873b45). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by wwmayer** [comment]
Can you squash your commits to a single one, please?

**Comment by adrianinsaval** [comment]
a maintainer can probably edit it while merging but that is not a good commit message, if you where to look just at it "unification of changes" does not convey anything of it's content. Something like the PR title would be better.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842">pipeline status</a> for feature branch [PR_8842](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8842). Pipeline [803352574](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/803352574) was triggered at [8482fde](https://github.com/FreeCAD/FreeCAD/pull/8842/commits/8482fdeac42708328e6bbff2fdd09138ee974730). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by donovaly** [comment]
Looks OK now for me.

**Comment by luzpaz** [comment]
```shell
git commit --amend -m "Removal of unused code due to old <5.2 Qt library"
git push
```

**Comment by wwmayer** [comment]
Using the option Squash & Merge did the trick. There I could change the commit message.

**Pull request merged**